src/qt/bitcoingui.cpp

```diff
@@ -1104,8 +1103,6 @@ void BitcoinGUI::setNumBlocks(int count, const QDateTime& blockDate, double nVer
         tooltip += tr("Last received block was generated %1 ago.").arg(timeBehindText);
         tooltip += QString("<br>");
         tooltip += tr("Transactions after this will not yet be visible.");
-    } else if (fLiteMode) {
```

Comment:
This should be kept

Reply:
8ebbf9b1ec90048646492a47a93c14c8674929dc